developers/weaviate/configuration/replication.md

```diff
@@ -8,21 +8,21 @@ import Badges from '/_includes/badges.mdx';
 
 <Badges/>
 
-:::info Related pages
+:::info Prerequisites
```

Comment:
Not sure about this, given that it's a reference guide and not a linear tutorial. I think I prefer `Related Pages.`

Reply:
I made that chjange because the Configuring Replication page used concepts only explained in the Replication Architecture pages. "Related pages" wouldn't nudge me to read those right now. "Prerequisites" tells me that if I don't understand something, it's because I haven't read the prereqs.